### PR TITLE
Update free-courses-hi.md

### PR DESCRIPTION
**Pull request by itsmeaadi1**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

### Description
Added AWS course in hindi
### Why is this valuable (or not)?
It teaches hindi speakers how to use aws
### How do we know it's really free?
it is a youtube video
## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes

**Comment by itsmeaadi1** [comment]
Done with the changes

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -195,6 +195,7 @@
 #### AWS
 
 * [AWS Tutorial for Beginners In Hindi](https://www.youtube.com/playlist?list=PLCFe3TcoBniI4iaavlfnR0UGQrtjgOMj9) - Together with Abhi
+* [AWS Tutorial in HINDI](https://www.youtube.com/playlist?list=PL_OdF9Z6GmVZCwyfd8n6_50jcE_Xlz1je) - S3CloudHub
```
Comment (eshellman) [inline courses/free-courses-hi.md:198]:
```suggestion
* [AWS Tutorial in Hindi](https://www.youtube.com/playlist?list=PL_OdF9Z6GmVZCwyfd8n6_50jcE_Xlz1je) - S3CloudHub
```
no all caps

**Comment by itsmeaadi1** [comment]
Done

**Pull request merged**